manifests/init.pp

```diff
@@ -105,6 +105,14 @@
 # $autosign_mode::                          mode of the autosign file/script
 #                                           type:Pattern[/^[0-9]{4}$/]
 #
+#
+# $autosign_content::                       If set, write the autosign file content
```

Comment:
Sure, does it matter that this argument can be overloaded with a string or file() or template()?